synapse/tests/utils.py

```diff
@@ -825,6 +825,10 @@ def raises(self, *args, **kwargs):
         '''
         return self.assertRaises(*args, **kwargs)
 
+    async def asyncraises(self, exc, coro):
```

Comment:
can we get an example of this in `synapse.tests.test_utils` ?

Reply:
done